### PR TITLE
Fixed code to return all products

### PR DESCRIPTION
**Pull request by rcastley**

### CODE REVIEW

File: Google Docs Dashboard/DocsDashboard.js [hunk 1]
```diff
@@ -1,54 +1,61 @@
...
- 
-    /*Build and issue the Request URL, starts with base URL, and then adds your key and any optional parameters*/
-    var api_call = UrlFetchApp.fetch(api_url + "?api_key=" + api_key + "&format=xml&type=list").getContentText();
-   
-    //Parse the XML and place the spreadsheet into a container
-    var xml = Xml.parse(api_call, true), sheetData = SpreadsheetApp.getActiveSpreadsheet();
-   
-    //Call the function to create the spreadsheet headers
-    setSheetHeaders(sheetData);
+    var api_key = '8a4bae76-8992-3a8d-b826-17e87ed0224f';
```
Comment (iwithrow) [inline Google Docs Dashboard/DocsDashboard.js]: This should be 

/_insert your API Key here_/
var api_key = 'xxxxx';

**Comment by iwithrow** [comment]
I think this needs one minor edits, it shouldn't embed an active API key by default

**Comment by rcastley** [comment]
Updated the code to reflect the change.

**Comment by rcastley** [comment]
Re-opened issue so new code can be used.

**Pull request merged**